src/lib/ccf/index.ts

```diff
@@ -0,0 +1,458 @@
+import {INSTANCE_TYPE_COMPUTE_PROCESSOR_MAPPING} from '@cloud-carbon-footprint/aws/dist/lib/AWSInstanceTypes';
+import {IImpactModelInterface} from '../interfaces';
+import Spline from 'typescript-cubic-spline';
+import * as aws_instances from './aws-instances.json';
+import * as gcp_instances from './gcp-instances.json';
+import * as gcp_use from './gcp-use.json';
+import * as aws_use from './aws-use.json';
+import * as azure_use from './azure-use.json';
+import * as azure_instances from './azure-instances.json';
+import * as gcp_embodied from './gcp-embodied.json';
+import * as aws_embodied from './aws-embodied.json';
+import * as azure_embodied from './azure-embodied.json';
+import {KeyValuePair} from '../../types/boavizta';
+
```

Comment:
you can brake down imports part like this
```
import {INSTANCE_TYPE_COMPUTE_PROCESSOR_MAPPING} from '@cloud-carbon-footprint/aws/dist/lib/AWSInstanceTypes';
import Spline from 'typescript-cubic-spline';

import {IImpactModelInterface} from '../interfaces';

import * as aws_instances from './aws-instances.json';
import * as gcp_instances from './gcp-instances.json';
import * as gcp_use from './gcp-use.json';
import * as aws_use from './aws-use.json';
import * as azure_use from './azure-use.json';
import * as azure_instances from './azure-instances.json';
import * as gcp_embodied from './gcp-embodied.json';
import * as aws_embodied from './aws-embodied.json';
import * as azure_embodied from './azure-embodied.json';

import {KeyValuePair} from '../../types/boavizta';
```

to be logically separated like first goes native modules, then npm modules, interfaces, libs, utils, config, types

Reply:
btw we can present json file imports in UPPERCASE manner to make them different from our local variables since when they are in lowercase, it seems like we are not keeping camelcase convention 

AWS_INSTANCES, GCP_INSTANCES, etc.